web.Dockerfile

```diff
@@ -21,11 +21,13 @@ RUN swift build \
 # Run image
 # ================================
 FROM vapor/ubuntu:18.04
-WORKDIR /run
+WORKDIR /app
+
+ENV PORT 80
```

Comment:
Port 80 is not suitable for a general-purpose `Dockerfile` - it won't be bindable on the host side unless the user _running the container_ is `root` (and I expect the container would also need to be `--privileged`, though that's just a guess). It should absolutely be an `ENV`, for sure - in fact, it should be an `ARG` as well. The `Dockerfile` reference docs suggest a specific form:
```dockerfile
ARG PORT
ENV PORT ${PORT:-8080}
```
This allows the default to be overridden as a build arg during build and then again at execute time as an environment variable, as desired in both cases. The default value (if not set as an arg) gets embedded in the built image.

Reply:
(Side note - If I've misunderstood and this `Dockerfile` is actually intended for a less general deployment scenario, then 80 is absolutely a great default.)

Reply:
Adding the ARG definitely makes sense, will do.

I don't know where port 80 came from. I think it's a horrible idea, even though it seems --privileged is not needed, "just" forcing the app to run as root – which is also the default – is enough. Shall I change it to the regular 8080?